http-proxy/main.go

```diff
@@ -470,12 +470,14 @@ func main() {
 		PsmuxAggressivePadding:             *psmuxAggressivePadding,
 		PsmuxAggressivePaddingRatio:        *psmuxAggressivePaddingRatio,
 		BroflakeAddr:                       *broflakeAddr,
-		BroflakeCert:                       os.Getenv("BROFLAKE_CERT"),
-		BroflakeKey:                        os.Getenv("BROFLAKE_KEY"),
-		AlgenevaAddr:                       *algenevaAddr,
-		WaterAddr:                          *waterAddr,
-		WaterWASM:                          *waterWASM,
-		WaterTransport:                     *waterTransport,
+
+		// TODO (allan): Remove BroflakeCert and BroflakeKey once broflake is working properly on LC
+		BroflakeCert:   os.Getenv("BROFLAKE_CERT"),
+		BroflakeKey:    os.Getenv("BROFLAKE_KEY"),
```

Comment:
Sorry, I don't understand why we would want to leave these around.  The fields are not referenced anymore and there will never be anything in those environment variables at this point.